lib/pbench/server/database/models/users.py

```diff
@@ -51,14 +63,16 @@ def get_protected():
         return ["registered_on", "id"]
 
     @staticmethod
-    def query(id=None, username=None, email=None):
+    def query(id=None, username=None, email=None, all=False):
```

Comment:
Type hints?...particularly, the return type, please?

Reply:
Also, where is this new parameter used?  (I failed to find it.)

Reply:
It is used in the CLI for `user_list` command